src/Api/Program.cs

```diff
@@ -1,29 +1,80 @@
 using Api.Middlewares;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.IdentityModel.Tokens;
 using Domain.Options;
 using Infra.DB;
 using Infra.Repositories;
 using Microsoft.EntityFrameworkCore;
 using Service;
-
+using System.Text;
+using Application.Services;

```

Comment:
Se for dar merge na PR anterior, atualizar da main após.
Senão adequar o namespace para Service.